erts/preloaded/src/erts_internal.erl

```diff
@@ -484,6 +485,49 @@ perf_counter_unit() ->
 is_system_process(_Pid) ->
     erlang:nif_error(undefined).
 
+-spec sys_proc_outstanding_requests_limit() -> pos_integer().
+
+sys_proc_outstanding_requests_limit() ->
+    erlang:nif_error(undefined).
+
+set_code_and_literal_cleaner_prio(Prio) ->
+    Ref1 = make_ref(),
+    erts_code_purger ! {change_prio, self(), Ref1, Prio},
+    Ref2 = make_ref(),
+    LAC = find_lac(),
+    LAC ! {change_prio, self(), Ref2, Prio},
+    [{code_purger, receive {Ref1, OP1} -> OP1 end},
+     {literal_area_collector, receive {Ref2, OP2} -> OP2 end}].
+
+find_lac() ->
```

Comment:
I'm sure there is a reason, but why isn't the `lac` registered?

Reply:
It should not be directly addressed by anyone except for the VM itself which have a direct pointer to the process structure. This function is only there as a fallback if we should need to reduce the priority at a customer site if something unexpected should occur now when this functionality is newly introduced. I expect to remove this functionality in OTP 25.

Reply:
I removed the premature optimization in `find_lac()` all together.